packages/reqresp/test/fixtures/messages.ts

```diff
@@ -0,0 +1,168 @@
+import {Uint8ArrayList} from "uint8arraylist";
+import {createIBeaconConfig, IBeaconConfig} from "@lodestar/config";
+import {chainConfig} from "@lodestar/config/default";
+import {allForks, altair, phase0, ssz} from "@lodestar/types";
+import {fromHexString} from "@chainsafe/ssz";
+import {SLOTS_PER_EPOCH} from "@lodestar/params";
+import * as messagesDef from "../../src/messages/index.js";
+import {EncodedPayloadType, ProtocolDefinition, TypeSerializer} from "../../src/types.js";
+import {ZERO_HASH} from "../utils/index.js";
+
+type MessageFixture<T> = {
+  type: TypeSerializer<T>;
+  payload: {
+    type: EncodedPayloadType.ssz;
+    data: T;
+  };
+  chunks: Uint8ArrayList[];
+  asyncChunks: Buffer[];
+};
+
+export const sszSnappyPing: MessageFixture<phase0.Ping> = {
+  type: ssz.phase0.Ping,
+  payload: {
+    type: EncodedPayloadType.ssz,
+    data: BigInt(1),
+  },
+  chunks: ["0x08", "0xff060000734e61507059010c00000175de410100000000000000"].map(
+    (s) => new Uint8ArrayList(fromHexString(s))
+  ),
+  asyncChunks: [
+    "0x08", // length prefix
+    "0xff060000734e61507059", // snappy frames header
+    "0x010c00000175de410100000000000000", // snappy frames content
+  ]
+    .map(fromHexString)
+    .map(Buffer.from),
```

Comment:
I know it's a test, but let's not accept that doing two iterations instead of one is acceptable in Lodestar's codebase :pray: